codespell

```diff
@@ -0,0 +1 @@
+feild
```

Comment:
Are we able to inline comments here as to why these are necessary?

Reply:
Removing